src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -169,6 +169,12 @@ class NetworkOPsImp final
          * @return JSON object.
          */
         StateCountersJson json() const;
```

Comment:
nit: I think I saw that `json` member function could use the new `getCounterData`